tests/DCPS/Messenger/run_test.pl

```diff
@@ -37,10 +37,18 @@
     $pub_opts .= " -DCPSConfigFile pub_udp.ini";
     $sub_opts .= " -DCPSConfigFile sub_udp.ini";
 }
+elsif ($test->flag('udp_free')) {
```

Comment:
What is this one for?

Reply:
Ah, strictly speaking this doesn't need to be here. I added a test case to help me debug why "./run_test.pl udp" wasn't working on my machine but "./run_test.pl default_udp" was. After invesitaging, it turned out that the problem was that Ubuntu only maps "localhost" to 127.0.0.1 and not ::1 (which is "ip6-localhost") and I had ipv6 enabled. Once I added "localhost ::1" to my /etc/hosts file, the normal udp test worked as expected. I added the NetworkAddress test for it in a different PR, but kept this test here. I don't know that it adds much and I didn't add it to dcps_tests.lst, so if you want me to remove it I'm fine with that.

Reply:
It's fine to keep it, maybe just add a comment here or in the ini files.